src/common/checksum.c

```diff
@@ -1419,7 +1419,9 @@ ci_insert(struct dcs_csum_info *dcb, int idx, uint8_t *csum_buf, size_t len)
 {
 	uint8_t *to_update;
 
-	D_ASSERT(idx < dcb->cs_nr);
+	D_ASSERTF(idx < dcb->cs_nr, "idx(%d) < dcb->cs_nr(%d)",
```

Comment:
I presume these can't be caused by user error?

Reply:
Shouldn't be able to. All calls to this are internal to the server.